index.html

```diff
@@ -174,13 +303,54 @@ <h2>Multi-Post Stories</h2>
                   <li>css</li>
-                  <li class="js">javascript</li>
+                  <li>javascript</li>
                 </ul>
-            <form action="#">
-                <button class="btn-body btn" type="submit" name="submit">see project</button>
-            </form>
-          </div>
-      </section>
+                <button class="btn-body btn" id="myBtn4" type="button" name="submit">see project</button>
+              </div>
+              <div id="myModal4" class="modal">
+                <div class="modal-content">
+                    <div class="modal-title">
+                      <div class="toggle-header">
+                        <div class="toggle-text-top">
+                          <h1>Facebook 360</h1>
+                          <ul class="work-top work-top-toggle">
+                            <li class="canopy">CANOPY</li>
+                            <li><img class="counter" src="img/Counter.svg" alt="counter"></li>
+                            <li class="dev">back end dev</li>
+                            <li><img class="counter" src="img/Counter.svg" alt="counter"></li>
+                            <li class="dev">2015</li>
+                          </ul>
+                       </div>
+                        <span class="close4">&times;</span>
+                      </div>
+                      <img class="modal-img" src="img/Portfolio4-big.svg" alt="work-image">
+                    </div>
+                  
+                  <div class="toggle-body">
+                    <p class="toggle-p">
+                      Lorem ipsum dolor sit amet, consectetur adipisicing elit. A labore quibusdam non optio  itaque repellat inventore nisi, magnam accusamus tenetur quod. Sint ea sit similique   incidunt explicabo voluptatibus quae quia.  Lorem ipsum dolor sit amet consectetur  adipisicing elit. Sunt, delectus laudantium laboriosam animi voluptates eveniet quis   repellat porro cupiditate officiis quibusdam doloremque libero numquam facilis unde   laborum et exercitationem nihil.
+                    </p>
+                    <nav class=toggle-nav>
+                     <ul class="toggle-list-top">
+                       <li> <p>html</p></li>
+                       <li><p>css</p></li>
+                       <li><p>javascript</p></li>
+                     </ul> 
+                     <ul class="toggle-list-bottom">
+                       <li><p>github</p></li>
+                       <li><p>ruby</p></li>
+                       <li><p>boostrap</p></li>
+                     </ul>
+    
+                      <form action="#">
+                        <button type="button" name="submit" class="btn-toggle" >see live</button>
+                        <Button type="button" name="submit" class="btn-toggle">see source</Button>
+                      </form>
+                    </nav>
+                  </div>
+                </div>
+              </div>
+          </section>
```

Comment:
- [ ] From the projects requirements, I would like to advise you generate projects dynamically, so that it will be easy in the future to add new projects